oz/Guest.py

```diff
@@ -79,6 +79,12 @@ def _discover_libvirt_type(self):
 
         self.log.debug("Libvirt type is %s", self.libvirt_type)
 
+        # For backward compatibility: If using KVM, we always want the best CPU the host can offer
+        # as we don't need to worry about live migration portability
+        if self.libvirt_cpu_mode is None and self.libvirt_type == 'kvm':
+            self.log.debug("Set libvirt CPU mode to cpu-passthrough by default for type kvm")
+            self.libvirt_cpu_mode = 'cpu-passthrough'
```

Comment:
This `cpu_mode`=`cpu-passthrough` is invalid, the cpu_mode option can take one of the following values: none, host-passthrough, host-model, and custom.
`Image build FAILED with error: unsupported configuration: Invalid mode attribute 'cpu-passthrough'`

Reply:
Seems like a typo on my part, yes, I will abandon this as we are not using oz anymore.